content/posts/bootstrap-your-aurora-database-and-schemes-at-creation.md

```diff
@@ -212,7 +212,7 @@ function, and that's something you don't have to do within the web console.<br>
 You may want to retrieve the root user and password from the parameter store so
 they don't appear in clear text in your Lambda.<br>
 You can also see that we grab the Lambda code from an Osones public bucket to
-make your life easier, but for you, here is the code:
+make your life easier, here is the code if you don't want to rely on external sources:
```

Comment:
No, that's just to explain how the Lambda works, and so on show it. They still will rely on an external resource, mine.